HTMLView.js

```diff
@@ -14,6 +15,7 @@ const baseStyles = StyleSheet.create({
   i: italicStyle,
   em: italicStyle,
   u: underlineStyle,
+  s: strikethroughStyle,
```

Comment:
you should also add this for the deprecated `strike` tag, given that this code also recognizes the old `b` and `i` tags

Reply:
@TooManyBees 👍 

![](https://media3.giphy.com/media/1wQOxawz5RpJDg2CIB/giphy.gif)